packages/ember-views/lib/views/core_view.js

```diff
@@ -7,9 +7,9 @@ import ActionHandler, { deprecateUnderscoreActions } from 'ember-runtime/mixins/
 import { typeOf } from 'ember-runtime/utils';
 
 import { InteractiveRenderer } from 'ember-htmlbars/renderer';
+import DOMHelper from 'ember-htmlbars/system/dom-helper';
```

Comment:
why did you change this?  this was for a path only used by tests in ember-views we are working to get rid of.